source/wp-content/themes/wporg-themes-2024/src/utils/listbox.js

```diff
@@ -0,0 +1,201 @@
+/* global CustomEvent */
+/**
+ * WordPress dependencies
+ */
+import { DOWN, END, ENTER, HOME, LEFT, RIGHT, SPACE, UP } from '@wordpress/keycodes';
+
+class wporgListbox {
+	constructor( container, state ) {
+		this.state = state;
+		this.container = container;
+		this.selected = state.initialSelected > 0 ? state.initialSelected : null;
+		this.current = null;
```

Comment:
It's not clear to me what the difference is between these two; I'm gaining an understanding as I read through, but I think some doc comments would help